src/index.ts

```diff
@@ -1,88 +1,106 @@
-import { getDependencies } from './getDependencies';
+import { getDependencies, getDependenciesYaml } from './getDependencies';
 import { getRepoIds } from './getRepos';
-const JupiterOneClient = require('@jupiterone/jupiterone-client-nodejs');
-const gql = require('graphql-tag');
-const readlineSync = require('readline-sync');
+import { getClient } from './getClient';
+import { question } from 'readline-sync';
+import { createRepoRelationship, createDeployRelationship } from './createRepoRelationship';
 
 require('dotenv').config();
 
-let success = 0, failure = 0;
-const missingDeps = [];
+let success = 0, failure = 0, switchName;
+let missingDeps = [];
 let packageScope = [];
 
 (async () => {
   while (true) {
-    var scope = readlineSync.question('Please input a package scope, i.e. @package (input QUIT when finished): ');
+    const scope = question('Input a package scope, i.e. @package (input QUIT when finished, HELP for other options): ');
     if (scope === 'QUIT') {
       break;
     }
+    if (scope === 'HELP') {
+      console.log('');
+      console.log('1. ALL = All dependencies');
+      console.log('2. (No Input) = All dependencies not starting with \'@\'');
+      console.log('');
+      continue;
+    }
     packageScope.push(scope);
   }
-  console.log('');
-  const j1Client = await new JupiterOneClient({
-    account: 'j1dev',
-    username: 'j1dev',
-    password: 'dev',
-    poolId: process.env.J1_USER_POOL_ID,
-    clientId: process.env.J1_CLIENT_ID,
-    accessToken: process.env.J1_API_TOKEN,
-    dev: process.env.J1_DEV || false
-  }).init();
+  const pathToRepos = question('Input path to directory with repos (relative to root directory): ');
+  const account = question('Input account. If using a .env file, continue without input: ');
+  const username = question('Input username. If using a .env file, continue without input: ');
+  const password = question('Input password. If using a .env file, continue without input: ');
+  const access = question('Input access token. If using a .env file, continue without input: ');
+  const clientInput = {account: account, username: username, password: password, accessToken: access};
+
 
-  const repoMap = await getRepoIds();
+  console.log('');
+  const j1Client = await getClient(clientInput);
+  const repoMap = await getRepoIds(pathToRepos, clientInput);
+  console.log('');
```

Comment:
Just making sure, are these console logs intentional to make output look nicer?

Reply:
Yeah they are